CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT(binaryen CXX)
+PROJECT(binaryen C CXX)
```

Comment:
Unfortunately C is needed for support of old version of cmake module FindThreads. Now cmake miniumum version is 2.8.7, from 3.4.0 it does'nt need C.